web/src/components/Header.tsx

```diff
@@ -80,10 +80,10 @@ const ActionBar = styled.nav`
   justify-content: flex-end;
   gap: 12px;
   padding: 0 16px;
+  flex: 1 1 0;
```

Comment:
:question: Any reason why we use this instead of just `flex: 1;`?

Reply:
I believe the flex-shrink is necessary to not get things like this: 
<img width="1153" alt="image" src="https://github.com/user-attachments/assets/59d0d3ac-3131-41fb-8268-2b6807c8ec3e">

Reply:
Hmmm. I can't reproduce that on my device, I get the same result with both `flex: 1;` and `flex: 1 1 0;` and both look very different than this :see_no_evil: 
Also, according to MDN, the last property defines `flex-basis`.

Also, the current behavior still seems to be not optimal: Even though there is more than enough space, the title already wraps since the nav item takes a lot of space:
![image](https://github.com/user-attachments/assets/e08f2f63-ac64-4441-a662-bc8c9e09a294)

Could you perhaps investigate that?

Reply:
Ah, sorry, I thought you were asking about the `flex-shrink`. Though now that I think about it, I think that is `1` by default anyway... I'll take a look at what's going on with the smaller screen.